Gemfile

```diff
@@ -7,6 +7,7 @@ gem 'activerecord-session_store'
 gem 'google-api-client', '~> 0.7.1'
 
 gem 'sufia', '~> 6.2.0'
+gem 'jbuilder', '~> 2.0' # hidden sufia dependency
```

Comment:
Sufia appears to be using Jbuilder but they do not have it as a dependency :disappointed:  ? Am I missing something here? Without this in our Gemfile, creating a GF will fail from the UI.

PR for adding usuage of jbuilder: https://github.com/projecthydra/sufia/pull/797
Jbuilder file in sufia 6.2 that is causing the issue: https://github.com/projecthydra/sufia/blob/c9cbfdbd7b52619aaf848a3d3b9a984c452a0481/app/views/generic_files/jq_upload.json.jbuilder

Even in current Sufia, this problem appears to still exist (see curation concerns/sufia).